### PR TITLE
Configure libtiff with ZIP and JPEG.

### PR DESCRIPTION
**Pull request by GunnarFarneback**
It seems a bit sad to depend on Zlib and JPEG but not configure libtiff to enable them, so this PR does just that. Possibly some paths to those libraries will need to be configured as well, CI will tell.

### CODE REVIEW

**Comment by GunnarFarneback** [comment]
What?
```
WARNING: unrecognized options: --with-ZIP, --with-JPEG
```
Those were fine when running `.configure` locally.

File: L/Libtiff/build_tarballs.jl [hunk 1]
```diff
@@ -16,7 +16,7 @@ script = raw"""
 cd $WORKSPACE/srcdir/tiff-*/
 CPPFLAGS="-I${prefix}/include"
```
Comment (giordano) [inline L/Libtiff/build_tarballs.jl:17]: This needs to be exported
```suggestion
export CPPFLAGS="-I${prefix}/include"
```

**Comment by GunnarFarneback** [comment]
Well, it looks like http://www.libtiff.org/build.html is outdated.

From `configure.ac` it seems like ZLIB and JPEG are enabled by default but it requires that the configure script finds include files and library files in some standard location, which doesn't appear to be the case for the Yggdrasil builds. The locations can be set explicitly with `--with-zlib-include-dir=DIR` etc. though.

File: L/Libtiff/build_tarballs.jl (hunk 1 above)
Comment (GunnarFarneback) [reply L/Libtiff/build_tarballs.jl]: The build looks good with this fix, whereas my commit was entirely nonsensical based on outdated information.

**Comment by GunnarFarneback** [comment]
I rebased out my commit since it was definitely incorrect and the CPPFLAGS export seems effective.

**Comment by GunnarFarneback** [comment]
Btw, it looks like an awful lot of Xorg packages have the same CPPFLAGS mistake. Didn't find any other suspects.

**Comment by GunnarFarneback** [comment]
The build logs look good now, with ZLIB and JPEG properly configured. From my point of view this is ready to be merged.

**Comment by giordano** [comment]
> it looks like an awful lot of Xorg packages have the same CPPFLAGS mistake

They were very likely written by the same person :innocent:

I'm moderately sure that the header files are found in other ways (maybe pkgconfig files?) in the Xorg builders, otherwise they wouldn't work at all.

**Pull request merged**